source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,75 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Comparison Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use comparison operations to compare two values. The methods
+described in this section perform a comparison operation on ``MqlValue``
+objects.
+
+The following table lists the comparison methods
+available in the driver and corresponding expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `eq() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#eq(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$eq </reference/operator/aggregation/eq/>`
+
+   * - `gt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#gt(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$gt </reference/operator/aggregation/gt/>`
+
+   * - `gte() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#gte(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$gte </reference/operator/aggregation/gte/>`
+
+   * - `lt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#lt(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$lt </reference/operator/aggregation/lt/>`
+
+   * - `lte() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#lte(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$lte </reference/operator/aggregation/lte/>`
+
+   * - `ne() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#ne(com.mongodb.client.model.mql.MqlValue)>`__
+     - :manual:`$ne </reference/operator/aggregation/ne/>`
```

Comment:
I: It seems a little strange (and less than ideal, from a usefulness perspective) to have all of the Java methods link to the same API page--especially when I couldn't find some of the methods I was looking for there.

S: Have each method link directly to its documentation in the API. E.g., here's eq():

https://mongodb.github.io/mongo-java-driver/4.9/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#eq(com.mongodb.client.model.mql.MqlValue)

Reply:
Weird - I don't know how that happened. If you look at the text, I had links to each one individually - or so I thought. I must have just copied the # and not noticed that I it was `MqlNumber` instead of `MqlValue`. Nice catch!